spec/requests/create_dro_spec.rb

```diff
@@ -218,6 +219,56 @@
     end
   end
 
+  context 'when the file is application/x-stanford-json' do
+    let(:blob) do
+      ActiveStorage::Blob.create!(key: 'tozuehlw6e8du20vn1xfzmiifyok',
+                                  filename: 'file2.txt', content_type: 'application/x-stanford-json',
+                                  byte_size: 10, checksum:)
+    end
+    let(:expected_content_type) { 'application/json' }
+
+    it 'switches the content type to application/json' do
+      post '/v1/resources',
+           params: request,
+           headers: { 'Content-Type' => 'application/json', 'Authorization' => "Bearer #{jwt}" }
```

Comment:
not sure either way, but would it be more like actual usage to have the `Content-Type` header be our `application/x-stanford-json` custom type?  or will sdr-client send across `application/json` for this request?

Reply:
I believe this `application/json` stays as is since that is the format of wrapper data being posted, and the `request` blob posted is the one with `application/x-stanford-json`

Reply:
oh, yeah, i think you're right!  sorry, thanks for the explanation.